### ISSUE TITLE
Support for SSL

### ISSUE THREAD

**Issue by mubix** [dirkjanm/ldapdomaindump#6]
Currently errors out if SSL is required

**Comment by mubix** [comment]
Changing line 757 to look like this:
```
s = Server(args.host, port=636, use_ssl=True, get_info=ALL)
```
Worked as a temp fix

**Comment by mubix** [comment]
putting a simple flag in to enable SSL and a check for the error would be awesome, sorry I can't provide the error :/

**Comment by dirkjanm** [comment]
Did you try to specify the host with ldap**s**://hostname (or IP)? Then it will automatically switch to use SSL.

**Comment by dirkjanm** [comment]
Assuming the fix in b905d9c makes SSL support somewhat more clear, closing.
